inst/shiny/modules/tab_tlg.R

```diff
@@ -1,96 +1,83 @@
+.TLG_DEFINITIONS <- {
+  defs <- yaml::read_yaml(system.file("shiny/tlg.yaml", package = "aNCA"))
+
+  defs <- purrr::imap(defs, \(opt_def, opt_id) {
+    if ("template" %in% names(opt_def)) {
+      template_def <- defs[[opt_def$template]]
+
+      for (d in names(opt_def)) {
+        if (d == "template") next
+
+        if (d == "options") {
+          for (o in names(opt_def$options)) {
+            template_def$options[[o]] <- opt_def$options[[o]]
+          }
+        } else {
+          template_def[[d]] <- opt_def[[d]]
+        }
+      }
+
+      opt_def <- template_def
+    }
+
+    opt_def
+  }) |>
+    setNames(defs)
+}
+
 tab_tlg_ui <- function(id) {
   ns <- NS(id)
 
-  tabsetPanel(
-    tabPanel(
+  navset_pill(
+    id = ns("tlg_tabs"),
+    nav_panel(
       "Order details",
-      actionButton(ns("add_tlg"), "Add TLG"),
-      actionButton(ns("remove_tlg"), "Remove TLG"),
-      actionButton(ns("submit_tlg_order"), "Submit Order Details"),
-      DTOutput(ns("selected_tlg_table")),
-      actionButton(ns("submit_tlg_order_alt"), "Submit Order Details")
-    ),
-    tabPanel(
-      "Tables",
-      fluidRow(
-        column(
-          2,  # Left column for plot selection
-          radioButtons(
-            inputId = ns("buttons_Tables"),
-            label = "Choose Table\n",
-            choices = ""
-          )
-        ),
-        column(
-          6,  # Middle column for plot output
-          h4("Table to display"),
-          plotOutput(ns("plot_Tables"))
-        ),
-        column(
-          2,  # Right column for plot customization inputs
-          h4("Inputs with selected vals linked to downloadable obj (i.e, tlg_order())"),
-          textInput(ns("footnote_Tables"), label = "Footnote")
+      card(
+        style = "margin-top: 1em;",
```

Comment:
suggestion: i noticed recently that ppl use `htmltools::css()`. probably it makes R folks more comfortable reading this stuff. just FYI, not actionable.